src/main/java/com/rabbitmq/perf/Consumer.java

```diff
@@ -112,7 +113,7 @@ public void run() {
         } catch (IOException e) {
             throw new RuntimeException(e);
         } catch (InterruptedException e) {
-            throw new RuntimeException(e);
+            LoggerFactory.getLogger(getClass()).warn("Consumer thread has been interrupted");
```

Comment:
If I understand correctly, InterruptedException is silenced here, though we log a message. Is it intentional? Is it part of the refactoring for Java 8?

Reply:
It's not part of the refactoring for Java 8. Some of the new tests interrupt the consumer threads and this exception stack trace would end up in the console. Now there's a warning and the thread simply ends, without polluting the console. This is one way to handle the exception without too much fuss.

Reply:
Ok, thanks for the explanation.